frontend/src/app/[locale]/subscribe/page.tsx

```diff
@@ -54,12 +51,7 @@ export default function Subscribe() {
             })}
           </Grid>
           <Grid tabletLg={{ col: 6 }}>
-            <NextIntlClientProvider
```

Comment:
had to make this change to get tests to work, but this wasn't doing anything anyway